include/crow/routing.h

```diff
@@ -1813,6 +1813,11 @@ namespace crow // NOTE: Already documented in "crow/app.h"
             {
                 throw;
             }
+            catch (const bad_request& e)
+            {
+                res = response (400);
+                res.body = e.what();
+            }
```

Comment:
I think the include for `exceptions.h` is missing and it's causing the tests to fail.